src/xo-app/vm/tab-disks.js

```diff
@@ -64,6 +64,110 @@ const parseBootOrder = bootOrder => {
   forEach(bootOptions, (text, id) => { order.push({id, text, active: false}) })
   return order
 }
+const COLUMNS = [
+  {
+    itemRenderer: (vdi, userdata) => {
+      return <Text value={vdi.name_label}
+        onChange={value => editVdi(vdi,
+                { name_label: value })} />
+    },
+    default: true,
+    name: _('vdiNameLabel'),
+    sortCriteria: _ => _.name_label
+  }, {
+    itemRenderer: (vdi, userdata) => {
+      return <Text value={vdi.name_description} onChange={value => editVdi(vdi, { name_description: value })} />
+    },
+    default: true,
+    name: _('vdiNameDescription'),
+    sortCriteria: _ => _.name_description
+  },
+  {
+    itemRenderer: (vdi, userdata) => {
+      return <Size value={vdi.size || null} onChange={size => editVdi(vdi, { size })} />
+    },
+    default: true,
+    name: _('vdiSize'),
+    sortCriteria: _ => _.size
+  }, {
+    itemRenderer: (vdi, userdata) => {
+      const srs = userdata[2]
+      const vm = userdata[3]
+      const sr = srs[vdi.$SR]
+      if (!sr) return
+      return <XoSelect
+        onChange={sr => migrateVdi(vdi, sr)}
+        xoType='SR'
+        predicate={sr => sr.$pool === vm.$pool && isSrWritable(sr)}
+        labelProp='name_label'
+        value={sr}
+        useLongClick
+                      >
+        <Link to={`/srs/${sr.id}`}>{sr.name_label}</Link>
+      </XoSelect>
+    },
+    default: true,
+    name: _('vdiSr'),
+    sortCriteria: sr => sr.name_label
+  },
+  {
+    itemRenderer: (vdi, userdata) => {
+      const vdis = userdata[0]
+      const vbds = userdata[1]
+      const vbd = vbds.filter((Vbd) => {
+        if (vdis[Vbd.VDI] === vdi) {
+          return vbd
```

Comment:
Although this would most likely work (if `vbd` isn't a falsy value), `filter`'s callback should return a boolean because it's mote semantically correct.